### PR TITLE
Build Linux tarball binary with support for ALSA backend

### PR DESCRIPTION
**Pull request by mikedickey**
Fixes also a bug with compression of Linux manpage

Also printing backend names now instead of RtAudio API ids

### CODE REVIEW

**Review by nwang92 — APPROVED** [review]

File: .github/workflows/jacktrip.yml [hunk 1]
```diff
@@ -462,9 +450,6 @@ jobs:
           if [[ "${{ matrix.weakjack }}" == true ]]; then 
             CONFIG_STRING="-Dweakjack=true $CONFIG_STRING"
           fi
-          if [[ "${{ runner.os }}" == 'Linux' ]]; then
-            CONFIG_STRING="-Dwerror=true $CONFIG_STRING"
-          fi
```
Comment (ntonnaett) [inline .github/workflows/jacktrip.yml:465]: I added this so we don't regress with code quality (at least on Linux). What warning is the reason for the removal?

Comment (mikedickey) [reply .github/workflows/jacktrip.yml:465]: RtAudio generates warnings when the ALSA backend is enabled. This caused the builds to fail.

Comment (mikedickey) [reply .github/workflows/jacktrip.yml:465]: Perhaps there is a way to do this without it also applying to subprojects?

Comment (ntonnaett) [reply .github/workflows/jacktrip.yml:465]: `-Drtaudio:werror=false` should work in our case I think.

Comment (mikedickey) [reply .github/workflows/jacktrip.yml:465]: Cool thanks, I'll give it a try!

Comment (ntonnaett) [reply .github/workflows/jacktrip.yml:465]: An alternative would be to add `include_type: 'system'` to the rtaudio dependency, like we did with Qt, but it has additional implications like ordering of build flags.

Comment (ntonnaett) [reply .github/workflows/jacktrip.yml]: Nice, looks like rtaudio:werror worked. :)

Comment (mikedickey) [reply .github/workflows/jacktrip.yml]: Looks like it worked!

**Review by nwang92 — APPROVED** [review]

**Pull request merged**